app/routes/docs.samples.tsx

```diff
@@ -0,0 +1,213 @@
+import { Link } from '@trussworks/react-uswds'
```

Comment:
Please reformat this file as Markdown. Given the large number of code samples, it will be much easier to maintain using GitHub Flavored Markdown style fenced code blocks, such as:

    ```python
     def foo():
        return 'bar'
    ```